care/facility/tasks/patient/discharge_report.py

```diff
@@ -0,0 +1,58 @@
+import datetime
+import random
+import string
+import time
+
+import celery
+from django.conf import settings
+from django.core.files.storage import default_storage
+from django.core.mail import EmailMessage
+from django.template.loader import render_to_string
+from django.utils.timezone import make_aware
+from hardcopy import bytestring_to_pdf
+
+from care.facility.models import DailyRound, PatientConsultation, PatientRegistration, PatientSample
+
+
+def randomString(stringLength):
+    letters = string.ascii_letters
+    return "".join(random.choice(letters) for i in range(stringLength))
```

Comment:
Codacy found an issue: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5765271)